docs/quick-answers/linode-platform/cloud-manager-search/index.md

```diff
@@ -13,80 +13,98 @@ contributor:
+The search tool can also be used with keywords to help filter and refine the results. Here is a list of common keywords and their definitions:
 
 | Keyword   | Definition   |
 |:--------------|:------------|
-| type | Type of resource. Linode, image, NodeBalancer etc...  |
-| tag  | Tag applied to your resources. See [here](/docs/quick-answers/linode-platform/tags-and-groups/) |
-| label | The label or identifier for your Linode  |
-| region | The region for your resources, such as `us-east`. For a full list, see the return values for our [API or CLI](https://developers.linode.com/api/v4/regions)  |
-| IP | The IP address of your resource  |
-| created | The date of your resource's creation |
-| status | Same values as the `status` field in our API. See our (API Documentation)[https://developers.linode.com/api/v4/linode-instances-linode-id] for all possible inputs. |
+| type | Type of resource. Linode, image, NodeBalancer, etc.  |
+| tag  | Tag applied to your resources. See the guide on [Tags and Groups](/docs/quick-answers/linode-platform/tags-and-groups/). |
+| label | The label or identifier for your Linode. |
+| region | The region for your resources, such as `us-east`. For a full list of regions, see the return values from the API or CLI [List Regions](https://developers.linode.com/api/v4/regions) endpoint. |
+| IP | The IP address of your resource. |
+| created | The date your resource was created. |
+| status | A brief description of a Linode's current state. See the API [View Linode](https://developers.linode.com/api/v4/linode-instances-linode-id) endpoint status field for all possible values. |
 
-Syntax for using keywords will always be the keyword itself, appended by a colon, and then a string defined by the keyword to limit your search. For example, to search for all images on your account, you would enter the following:
+The syntax for using keywords is:
+
+    keyword:string defined by the keyword to limit your search
+
+For example, to search for all images on your account, you would enter the following:
 
     type:image
 
-And to find all applicable resources created in the year 2020:
+To find all applicable resources created in the year 2020:
 
     created:2020
 
-## Standard Boolean Operators
+You can make this more focused by combining the standard search with the keywords. For example, the following query searches for NodeBalancers with the label or tag ecommerce:
 
-The Cloud Manager includes support for standard boolean operators `OR` and `AND` which can help to filter the results of your search further. `OR` can be used to search for multiple different keywords in a single query  and `AND` will allow you to narrow your search more precisely for additional qualifiers. For example, in the following query, we'd like to search for ALL of the Linodes and Images on our account:
+    type:NodeBalancer ecommerce
 
-    type:linode OR type:image
+## Standard Logical Operators
```

Comment:
I grouped, OR, AND, and - all under Logical Operators instead of Boolean and Negate by itself since this is a thing in programming.